packages/flame_jenny/example/lib/components/dialogue_button.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:flame/components.dart';
+import 'package:flame/input.dart';
+import 'package:flutter/material.dart';
+import 'package:jenny_example/commons/commons.dart';
+
+class DialogueButton extends SpriteButtonComponent {
+  DialogueButton({
+    required Vector2 posit,
+    required String assetPath,
+    required String text,
+    required super.onPressed,
+    super.anchor = Anchor.center,
+  }) : super() {
+    position = posit;
+    _text = text;
+    _assetPath = assetPath;
+  }
+
+  late String _text;
+  late String _assetPath;
+
+  @override
+  Future<void> onLoad() async {
+    Sprite dButton = await Sprite.load(_assetPath);
+    button = dButton;
```

Comment:
```suggestion
    button = await Sprite.load(_assetPath);
```
No abbreviations are allowed in this repository